modules/ROOT/pages/dw-expression-watches-view-reference.adoc

```diff
@@ -0,0 +1,22 @@
+= Evaluate DataWeave Expression View Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Studio debugger can watch for selected DataWeave expressions in all active breakpoints. If the expression occurs at at any of the active breakpoints, the debugger evaluates the expression, then displays the result in the *Mule Debugger* view.
```

Comment:
```suggestion
The Studio debugger can watch for selected DataWeave expressions in all active breakpoints. If the expression occurs at any of the active breakpoints, the debugger evaluates the expression, then displays the result in the *Mule Debugger* view.
```
Why "can watch"? Does the user have to do something to enable this? Or does the debugger just "watch" on a regular basis?
Who selects the expressions?
Where does evaluation come into this?
Do we mean...:
```suggestion
Select DataWeave expressions and the Studio debugger watches for them in all active breakpoints. If a selected expression appears, the debugger evaluates its and displays the result in the *Mule Debugger* view:
```